readme.txt

```diff
@@ -116,6 +116,45 @@ You'll find the [FAQ on Yoast.com](https://yoast.com/wordpress/plugins/seo/faq/)
 
 == Changelog ==
 
+= 3.5.0 =
+
+Release Date:
+
+* Enhancements:
+    * Adds Flesch Reading tests for Dutch and German
+    * Added info about author archives links in theme to disable setting.
+    * Explicitly checks for public post type status when creating sitemap.
+    * Removes frequency and priority from sitemap.
+    * Improves sitemap "blocking files" notification.
+    * Improves sitemap generation for posts when dealing with a large number of posts.
+    * Improves reliability in some PHP configurations.
+    * Improves styling for notices below tabs.
+    * Adds @id fields to JSON LD output
+    * Adds table headings and labels to the bulk editor.
+    * Improves the accessibility of the bulk editor.
+    * Improves plugin loading for network.
+    * Don't show keyword filter in post list when keyword analysis is disabled.
+    * Improves message for settings import.
+    * Adds translations for notification counts.
+    * Makes upload image buttons translatable.
```

Comment:
both are correct, but since we never use `translateable`, but `translatable` is used in some places in the changelog, I would suggest to keep it like this :)